### ISSUE TITLE
Generate splatting from other context

### ISSUE THREAD

**Issue by OmiCron07** [cezarypiatek/MappingGenerator#13]
Can you generate the splatting with other scenarios like  :

From
![image](https://user-images.githubusercontent.com/31325575/37781567-524570f0-2dc7-11e8-8625-376c5c56ef6f.png)
To
![image](https://user-images.githubusercontent.com/31325575/37781646-7b28d502-2dc7-11e8-9b11-473d7ebee32c.png)

**Comment by cezarypiatek** [comment]
Personally, I'm not using this linq-syntax on the daily basis. In "normal" constructor invocation the splatting should work but there must be some differences in linq syntax. I have to check this.

**Comment by OmiCron07** [comment]
You are totally right, just tested and when not in linq, it works. The linq is the culprit.

**Comment by cezarypiatek** [comment]
My fix seems to work in VS 15.6.6. @OmiCron07  could you confirm that with https://ci.appveyor.com/api/buildjobs/jn23alhhhsptqrbp/artifacts/MappingGenerator%2FMappingGenerator%2FMappingGenerator.Vsix%2Fbin%2FRelease%2FMappingGenerator.vsix ?
